src/modapi/server/modcreator.cpp

```diff
@@ -243,6 +243,262 @@ CModAPI_ModCreator::CModAPI_LineStyleCreator& CModAPI_ModCreator::AddLineStyle()
 	return LineStyle;
 }
 
+CModAPI_ModCreator::CModAPI_SkinModifierCreator& CModAPI_ModCreator::AddSkinModifier()
+{
+	int Id = m_SkinModifiers.size();
+	m_SkinModifiers.set_size(m_SkinModifiers.size()+1);
+	CModAPI_ModCreator::CModAPI_SkinModifierCreator &SkinModifier = m_SkinModifiers[Id];
+
+	SkinModifier.m_Id = Id;
+
+	return SkinModifier.SetVanillaSkin(0); // initialize everything to change nothing
+}
+
+CModAPI_ModCreator::CModAPI_SkinModifierCreator& CModAPI_ModCreator::CModAPI_SkinModifierCreator::SetVanillaSkin(const char *pName)
```

Comment:
Leave the json part for latter. This function should be named SetNoChange() or something like this.